source/docs/articles/sites/code/unsupported-modules-plugins.md

```diff
@@ -146,9 +151,8 @@ Also see [Multiple Servers + Batch Database Stream Wrapper (sandbox module)](htt
 **Issue**: TimThumb is no longer supported or maintained. See [https://code.google.com/p/timthumb/](https://code.google.com/p/timthumb/).
 <hr>
 
-### Revive Old Post
-**Issue**: Revive Old Post does not set a proper callback via OAuth and the Twitter module.  It attempts to use ["SERVER_NAME"] instead of the recommended ["HTTP_HOST"]. See [SERVER_NAME and SERVER_PORT on Pantheon](https://pantheon.io/docs/articles/sites/code/server_name-and-server_port/).
-
+### WordFence
+**Issue**: WordFence works, though activating Live Tracking will add a cookie that conflicts with Varnish.  See [https://wordpress.org/support/topic/wfvt-cookie?replies=5].
```

Comment:
@aeligature would disabling cookies within the plugin resolve the issue? If so, can you include a **Solution** statement here that links to the support topic?